src/core/lib/channel/promise_based_filter.h

```diff
@@ -124,6 +124,59 @@ class BaseCallData : public Activity, private Wakeable {
     }
   };
 
+  class Flusher {
+   public:
+    explicit Flusher(BaseCallData* call);
+    // Calls closures, schedules batches, relinquishes call combiner.
+    ~Flusher();
+
+    void Release(grpc_transport_stream_op_batch* batch) {
```

Comment:
Suggest calling this `Resume()` as well.

Reply:
done